app/models/user.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+class User < ApplicationRecord
+  has_many :posts
+  attr_accessor :remember_token
+  before_create :remember
```

Comment:
We need to remove this line of code. Trying to create a token for a user that is not created yet leads to an error.

Reply:
Thank you for your input but it is required in the [project](https://www.theodinproject.com/courses/ruby-on-rails/lessons/authentication) and if we don't add it somehow then the user won't have a token

![image](https://user-images.githubusercontent.com/52209067/73864630-a0ebcb80-4807-11ea-985f-7cc5340554e9.png)

so we changed the **remember function** so it works when creating a new user